Week3/homework/step2-5.js

```diff
@@ -1,11 +1,16 @@
 'use strict';
 
+// Test PASS
 function multiplyAll(arr) {
   // eslint-disable-next-line
-  let product = 1;
+    let product = 1;
 
-  // Replace this comment and the next line with your code
-  console.log(arr, product);
+  function oneTime(element) {
+    product = product * element;
+    return product;
```

Comment:
👍. However, returning `product` here is redundant as `forEach` doesn't really return anything.

Reply:
yes, your point is important. I have to read more about nested ForEach methods.